### ISSUE TITLE
Large /var/lib/dhcp/dhcpd.leases isn't fully parsed

### ISSUE THREAD

**Issue by bartlubberdink** [webmin/webmin#939]
We have a /var/lib/dhcp/dhcpd.leases file with aprox 1500 leases in it.
but webmin will only show 539 leases when running "list active leases"

is there any way to debug this? or is this just some file parsing limit?

**Comment by jcameron** [comment]
No, the problem is the " in the middle of the string. Should't there be a \ before it?

**Comment by bartlubberdink** [comment]
perhaps.. 
This line was added by the dhcpd server.
when we manualy removed the space. the full file was parsed correctly.
